src/Features/Core/Portable/OrganizeImports/OrganizeImportsService.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.CodeAnalysis.OrganizeImports
 {
-    internal static partial class OrganizeImportsService
+    public static partial class OrganizeImportsService
```

Comment:
I think if we follow the naming convention around Formatt*er* and Renam*er* is this becomes the ImportsOrganiz*er*. If nothing else it probably shouldn't have the "Service" suffix because that implies it's implementing IOrganizeImportsService.